packages/react-reconciler/src/ReactFiberCommitWork.new.js

```diff
@@ -323,7 +323,11 @@ function commitBeforeMutationLifeCycles(
   );
 }
 
-function commitHookEffectListUnmount(tag: HookEffectTag, finishedWork: Fiber) {
+function commitHookEffectListUnmount(
+  tag: HookEffectTag,
+  finishedWork: Fiber,
+  nearestMountedAncestor: Fiber | null,
```

Comment:
I'm adding this as a new param (rather than making this function call `finishedWork.return`) because `commitHookEffectListUnmount` and `commitHookEffectListUnmount2` will eventually be merged– and then passive effects will be involved as well.